docs/autotools.rst

```diff
@@ -22,6 +20,33 @@ If you use a (binary oriented) GNU/Linux distribution, `autoreconf` may
 be part of the `autoconf` package. In addition you may have to install
 `automake` and/or `pkg-config`, too.
 
+GNU/Linux distributions
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+Before running ./autogen.sh, install some packages.
+
+On Debian-based systems, do::
+
+    $ sudo apt install \
+        gcc make \
+        pkg-config autoconf automake \
+        python3-docutils \
```

Comment:
To install on ubuntu 18.04 I needed python-docutils, not python3-docutils, at least when installing with linuxbrew.

Error I got:
```
==> Installing universal-ctags from universal-ctags/universal-ctags
==> Cloning https://github.com/universal-ctags/ctags.git
Updating /home/lazarus/.cache/Homebrew/universal-ctags--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at c77db55 Merge pull request #2182 from ksamborski/bugfix/issue-2180
==> Downloading https://files.pythonhosted.org/packages/84/f4/5771e41fdf52aabebbadecc9381d11dea0fa34e4759b4071244fa094804c/docutils-0.14.tar.gz
Already downloaded: /home/lazarus/.cache/Homebrew/downloads/082747f2931278a960879f30afd38989c894b92d04eda36ead03e57dfb6703fc--docutils-0.14.tar.gz
==> python -c import setuptools... --no-user-cfg install --prefix=/home/linuxbrew/.linuxbrew/Cellar/universal-ctags/HEAD-c77db55/libexec/vendor --install-scripts=/home/linuxbrew/.linuxbrew/Cellar/universal-ctags/HEAD-c77db55/libexec/vendor/bin --single-ver
==> ./autogen.sh
==> ./configure --prefix=/home/linuxbrew/.linuxbrew/Cellar/universal-ctags/HEAD-c77db55
==> make
Last 15 lines from /home/lazarus/.cache/Homebrew/Logs/universal-ctags/04.make:
  File "/usr/bin/rst2man", line 21, in <module>
    from docutils.core import publish_cmdline, default_description
  File "/home/linuxbrew/.linuxbrew/Cellar/universal-ctags/HEAD-c77db55/libexec/vendor/lib/python2.7/site-packages/docutils/__init__.py", line 112, in <module>
    class ApplicationError(StandardError):
NameError: name 'StandardError' is not defined
Makefile:5754: recipe for target 'man/ctags-incompatibilities.7' failed
make[2]: *** [man/ctags-incompatibilities.7] Error 1
Makefile:5752: recipe for target 'man/ctags.1' failed
make[2]: *** [man/ctags.1] Error 1
make[2]: Leaving directory '/tmp/universal-ctags-20190825-23059-1uqb31g'
Makefile:4667: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/tmp/universal-ctags-20190825-23059-1uqb31g'
Makefile:1285: recipe for target 'all' failed
make: *** [all] Error 2
```